docs/guides/waffle-testing.md

```diff
@@ -125,6 +125,23 @@ expect(await greeter.greet()).to.equal("Hola, mundo!");
 
 We can modify the state of a contract in the same way we read from it. Calling `setGreeting` will set a new greeting message. After the `Promise` is resolved, we perform another assertion to verify that the greeting change took effect.
 
+### Running tests in parallel
```

Comment:
I'd move this to its own guide, to make it more visible.

Also, this is not a waffle thing.

Reply:
Done.